packages/dashboards/addon/components/navi-widget.ts

```diff
@@ -93,6 +93,13 @@ export default class NaviWidget extends Component<NaviWidgetArgs> {
     return isSuccessful ? arr(value as WidgetData) : null;
   }
 
+  /**
+   * whether or not there is data to display
+   */
+  get hasNoData() {
+    return this.data?.firstObject?.response?.meta.pagination?.numberOfResults === 0;
```

Comment:
I'm not sure `numberOfResults` will always be present. Are we populating this in Elide responses? We might be better off testing `response.rows.length`

Reply:
Sure we can use row length - do you want me to update report-view so that it also uses row length? https://github.com/yavin-dev/framework/blob/master/packages/reports/addon/components/report-view.ts#L92

Reply:
*nod*